tests/foreman/ui/test_host.py

```diff
@@ -1493,14 +1497,43 @@ def test_global_registration_with_gpg_repo_and_default_package():
         4. open the global registration form and update the gpg repo and key
         5. check host is registered successfully with installed same package
         6. check gpg repo is exist in registered host
-
-    :CaseAutomation: ManualOnly
     """
+    client = rhel7_contenthost
```

Comment:
I don't think it's necessary. You can just use `rhel7_contenthost.execute(cmd)` . Not a blocking comment.

Reply:
I just thought `client` would make more sense as we are registering the clients